helm/cloud-provider-vsphere/charts/vsphere-csi-driver/values.yaml

```diff
@@ -1,59 +1,59 @@
-config:
-  clusterId: ""
-  vcenter: ""
-  username: ""
-  password: ""
-  datacenter: ""
-  region: ""
-  zone: ""
-  thumbprint: ""
-
-controllerDeployment:
-  replicas: 1
-  csiAttacher:
-    image: k8s.gcr.io/sig-storage/csi-attacher
-    tag: v3.5.0
-  csiResizer:
-    image: k8s.gcr.io/sig-storage/csi-resizer
-    tag: v1.5.0
-  csiController:
-    image: gcr.io/cloud-provider-vsphere/csi/release/driver
-    tag: v2.7.1
-  livenessProbe:
-    image: k8s.gcr.io/sig-storage/livenessprobe
-    tag: v2.7.0
-  syncer:
-    image: gcr.io/cloud-provider-vsphere/csi/release/syncer
-    tag: v2.7.1
-  csiProvisioner:
-    image: k8s.gcr.io/sig-storage/csi-provisioner
-    tag: v3.2.1
-  csiSnapshotter:
-    image: k8s.gcr.io/sig-storage/csi-snapshotter
-    tag: v6.0.1
-
-nodeDaemonset:
-  nodeDriverRegistrar:
-    image: k8s.gcr.io/sig-storage/csi-node-driver-registrar
-    tag: v2.5.1
-  csiNode:
-    image: gcr.io/cloud-provider-vsphere/csi/release/driver
-    tag: v2.7.1
-  livenessProbe:
-    image: k8s.gcr.io/sig-storage/livenessprobe
-    tag: v2.7.0
-
-csiDriver:
-  attachRequired: true
-  podInfoOnMount: false
-
-storageClass:
-  enabled: true
-  delete:
-    isDefault: true
-    vcdStorageProfileName: "vSAN Default Storage Policy"
-    fileSystem: "ext4"
-  retain:
-    isDefault: false
-    vcdStorageProfileName: "vSAN Default Storage Policy"
-    fileSystem: "ext4"
+config:
+  clusterId: ""
+  vcenter: ""
+  username: ""
+  password: ""
+  datacenter: ""
+  region: ""
+  zone: ""
+  thumbprint: ""
+
+controllerDeployment:
+  replicas: 1
+  csiAttacher:
+    image: giantswarm/csi-attacher
```

Comment:
Isn't it better to refer docker.io explicitly? just asking to understand your choice.

Reply:
I like to be explicit, but at the same time I had impression from the honneybadger team that this way it would be easier to migrate from a container registry to another. You know if there is yet another variable called registry, we can prepend it to each image. I think vsphere-cluster has it this way for some image. But thinking about it again, I think it's better to be explicit here so I am changing it to `docker.io/...`